src/screens/Home.tsx

```diff
@@ -1,26 +1,125 @@
 import React from "react";
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, View, Button } from 'react-native';
-import { Text } from '../components/atoms/Text';
+import { StyleSheet, View, Button, Pressable, Text, Image } from 'react-native';
+import { LinearGradient } from "expo-linear-gradient";
+import { useFonts, Inter_500Medium, Inter_600SemiBold, Inter_700Bold } from '@expo-google-fonts/inter';
 
 export default function Home({ navigation }: { navigation: any }) {
+    const [fontsLoaded] = useFonts({
+        Inter_500Medium,
+        Inter_600SemiBold,
+        Inter_700Bold
+      });
+      
+      if (!fontsLoaded) {
+        // You can return a loading screen or a placeholder here until the font is loaded
+        return <View>
+            <LinearGradient
+                // Background Linear Gradient
+                colors={['rgb(121, 12, 172)', 'rgb(255,218,185)']}
+                style={styles.background}
+                start={{ x: 0.6, y: 0.25 }}
+                end={{ x: 1, y: 1 }}
+            />
+            <Text>Loading...</Text>
+            </View>;
+      }
     return (
-        <View style={styles.container}>
+        <View style={styles.body}>
+            <LinearGradient
+                // Background Linear Gradient
+                colors={['rgb(121, 12, 172)', 'rgb(35, 163, 181)']}
+                style={styles.background}
+                start={{ x: 0.6, y: 0.25 }}
+                end={{ x: 1, y: 1 }}
+            />
```

Comment:
Is this gradient meant to be across all the screens?

If so, maybe it could be an idea to make it a component.